code/core/src/controller/DefaultLayer.java

```diff
@@ -0,0 +1,12 @@
+package controller;
+
+public enum DefaultLayer {
+    LOW(10),
+    DEFAULT(20),
+    HIGH(30);
```

Comment:
Wäre es hier stimmiger, wenn wir von "Layern" sprechen, die Werte des Enums wie folgt umzubenennen?

```suggestion
    BOTTOM(10),
    DEFAULT(20),
    TOP(30);
```

Reply:
Gute Idee 👍🏻